.github/workflows/molecule.yml

```diff
@@ -13,9 +13,9 @@ jobs:
     runs-on: ubuntu-latest
 
     strategy:
-      # max-parallel: 4
+      max-parallel: 4
       matrix:
-        distro: [centos7, debian10, rockylinux8]
+        distro: [centos7, centos8, debian10, debian11, rockylinux8, rockylinux9, ubuntu2004, ubuntu2204]
```

Comment:
Looks like `rockylinux9` changed something with the way `dnf` handles repository-keys. Maybe we need to have a testing host to try this step "manually".

Reply:
Thanks!